package.json

```diff
@@ -76,10 +78,18 @@
     "posttest": "[ -z \"$TRAVIS\" ] || codecov",
     "codecov": "codecov",
     "bench": "BENCHMARK=true npm run test",
-    "version": "node -e \"console.log(require('./package.json').version)\""
+    "version": "node -e \"console.log(require('./package.json').version)\"",
+    "lint-staged": "lint-staged"
   },
   "dependencies": {
     "is-browser": "^2.0.1",
     "murmurhash-js": "^1.0.0"
-  }
+  },
+  "lint-staged": {
```

Comment:
No, since lint-staged will pass only staged files as an argument to the linter: https://github.com/okonet/lint-staged/blob/master/src/findBin.js#L7. For that reason it _must_ be run without paths as argument.